app-modules/prospect/src/Filament/Resources/ProspectResource/RelationManagers/CasesRelationManager.php

```diff
@@ -99,7 +100,14 @@ public function table(Table $table): Table
             ])
             ->headerActions([
                 CreateAction::make()
-                    ->modalHeading('Create new case'),
+                    ->modalHeading('Create new case')
+                    ->mutateFormDataUsing(function (array $data): array {
```

Comment:
Can you please explain this addition?

Reply:
Hmm. I think this should probably be in an observer. What do you think @Orrison?

Reply:
@danharrin I agree that I would make this fix in an observer for cases